roles/setup_repo/tasks/PG_RedHat_setuprepos.yml

```diff
@@ -149,12 +151,33 @@
   failed_when: >
     reposub.rc != 0 or 'error: ' in reposub.stdout.lower()
   changed_when: reposub.rc == '0'
-  when: >
-    tpa_subscription_token|length > 0
-    and install_bdr_packages|bool
-    and pg_version|int >= 14
+  when:
+    - enable_edb_repo|bool
+    - tpa_subscription_token|length > 0
+    - install_bdr_packages|bool
+    - pg_version|int == 14
```

Comment:
Is this only for pg_version == 14 or does it apply to 12-14? Are we planning on only supporting PGD on v14 and above?

Reply:
If `pg_version == 14` is accurate, then we can change the curl line to read `curl -sS "{{ edb_2q_base_repo_link }}/{{ item }}/14/rpm" | bash`

Reply:
We are currently not interested in supporting other versions for BDR4. I may consider discontinuing support for BDR 4 in the future. Our focus should be on supporting PGD 5 and later versions.